Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -814,11 +814,16 @@ def _exec_editor_test(self, request: _pytest.fixtures.FixtureRequest,
                 if has_crashed:
                     crash_output = editor_utils.retrieve_crash_output(run_id, workspace, self._TIMEOUT_CRASH_LOG)
                     test_result = Result.Crash(test_spec, output, return_code, crash_output, None)
+                    # Save the .dmp file which is generated on Windows only
+                    dmp_file_name = os.path.join(editor_utils.retrieve_log_path(run_id, workspace),
+                                                 'error.dmp')
+                    if os.path.exists(dmp_file_name):
```

Comment:
I assume that this doesn't throw an exception on non-Windows platforms, and just skips instead?

Reply:
Correct, if the file doesn't exist it will skip